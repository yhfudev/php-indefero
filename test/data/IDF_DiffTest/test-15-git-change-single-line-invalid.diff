diff --git a/test_file b/test_file
index 2134e75..2eaf9ea 100644
--- a/test_file
+++ b/test_file
@@ -1 +1 @@
-Steddy is awesome
+Steddy is very awesome