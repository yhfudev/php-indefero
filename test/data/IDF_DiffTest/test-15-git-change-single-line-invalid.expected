<?php return array (
  'test_file' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '',
          2 => "Steddy is awesome\n",
        ),
        1 => 
        array (
          0 => '',
          1 => '1',
          2 => 'Steddy is very awesome',
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => 1,
        ),
        1 => 
        array (
          0 => '1',
          1 => 1,
        ),
      ),
    ),
  ),
);